Exiled.Events/EventArgs/Player/PreAuthenticatingEventArgs.cs

```diff
@@ -85,10 +85,15 @@ public PreAuthenticatingEventArgs(string userId, ConnectionRequest request, int
         /// <summary>
         ///     Gets a value indicating whether the player can be authenticated or not.
         /// </summary>
-        public bool IsAllowed { get; private set; }
+        public bool IsAllowed { get; set; }
```

Comment:
```suggestion
        public bool IsAllowed { get; private set; }
```

It was already tested and fixed recently. The setter should be private - it's not possible to uncancel the preauth event. If it's cancelled, it's over, the player will be kicked (even if you uncancel the event).

Uncanceling leads to an malformed preauth response being sent to the client or/and redundant transport network layer operations, but the player will be kicked anyway.